libs/feature/editor/src/lib/record-form/record-form.component.ts

```diff
@@ -1,22 +1,40 @@
 import { CommonModule } from '@angular/common'
-import { ChangeDetectionStrategy, Component } from '@angular/core'
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  ViewChild,
+} from '@angular/core'
 import { UiInputsModule } from '@geonetwork-ui/ui/inputs'
 import { EditorFacade } from '../+state/editor.facade'
 import { EditorFieldState, EditorFieldValue } from '../models/fields.model'
+import { MatIconModule } from '@angular/material/icon'
 
 @Component({
   selector: 'gn-ui-record-form',
   templateUrl: './record-form.component.html',
   styleUrls: ['./record-form.component.css'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   standalone: true,
-  imports: [CommonModule, UiInputsModule],
+  imports: [CommonModule, UiInputsModule, MatIconModule],
 })
 export class RecordFormComponent {
+  @ViewChild('titleInput') titleInput: ElementRef
   fields$ = this.facade.recordFields$
 
   constructor(public facade: EditorFacade) {}
 
+  focusInput() {
+    this.titleInput.nativeElement.children[0].focus()
+  }
+
+  handleTitleChange(field: EditorFieldState, newTitle: string) {
```

Comment:
@LHBruneton-C2C when I was writing a test for this function I realized that this function does the same as `handleFieldValueChange`. `EditorFieldValue` can also be a string. So I think we can use `handleFieldValueChange` as well for the title. WDYT?

Reply:
If it's working this way, sure! Sharing the logic is the way to go. Can't remember why this didn't seem to work at first :/